Sources/Commands/PackageCommands/AddDependency.swift

```diff
@@ -50,33 +51,49 @@ extension SwiftPackageCommand {
         @Option(help: "Specify upper bound on the package version range (exclusive)")
         var to: Version?
 
+        @Option(help: "Specify dependency type (url, path, registry)")
```

Comment:
Drop the possible values from the help here, argument parser handles this.

Reply:
fixed